### DISCUSSION TITLE
Change Spotify account

### DISCUSSION THREAD

**Discussion by elfluna666** [KRTirtho/spotube#2171] [Q&A]
Does anyone know how to permanently disconnect your Spotify account so you can reconnect another one? I tried uninstalling but it remembers my old account. Thanks !

**Comment by Peperehobbits01** [comment]
Just click disconnect in the settings

**Comment by elfluna666** [comment]
It doesn't work. When I do it, I can't choose to log back in with another account.

**Reply by justaguydoinghisstuff** [reply]
On windows i solved it by going into `AppData\Roaming\oss.krtirtho\spotube\webview_window_Webview2\EBWebView` and deleting the `Default`  folder. This is the profile folder for the built in edge browser (webview) that it used to sign me in. Maybe deleting the full profile is overkill, you technically would only need to delete the cookies, but i have no idea where they are saved so i just deleted it all.


i dont know what device you are on but it could help.
